src/organization/actions/OpeningHoursSpecification/create-opening-hours-specification.action.ts

```diff
@@ -19,15 +20,28 @@ export const createOpeningHoursSpecification = async (
       }),
       data,
     );
-    if (response.status !== 201) {
-      throw errorFactory.create(
-        OrganizationErr.CREATE_OPENING_HOURS_SPECIFICATION_FAILED,
-      );
+    switch (response.status) {
+      case 201:
+        return response.data;
+      case 403:
+        throw errorFactory.create(
+          OrganizationErr.CREATE_OPENING_HOURS_SPECIFICATION_FORBIDDEN,
+        );
+      case 404:
+        throw errorFactory.create(
+          OrganizationErr.OPENING_HOURS_SPECIFICATION_NOT_FOUND,
```

Comment:
```suggestion
          OrganizationErr.ORGANIZATION_NOT_FOUND,
```

(se référer au swagger)